public/config.json

```diff
@@ -5,7 +5,7 @@
   "featureFlags": {
     "DISABLE_AUTHENTICATION": {
       "description": "Disable authentication flow when true",
-      "isActive": false,
+      "isActive": true,
```

Comment:
For me, it is not a regression. If I want to run the solution locally, I want to run it with authentication disabled. Which should be the default behavior out of the box?

Reply:
In my opinion, the default behavior should be to have the authentication enabled. Although this isn't going to generate a security issue by itself (the API will reject all requests since they are not authenticated), disabling the authentication flow should be something that the developer needs to do consciously, and not the behavior by default.